### PR TITLE
Github Action to prevent large files

### PR DESCRIPTION
**Pull request by Wumpf**
image

Part of #1403 

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [ ] I've included a screenshot or gif (if applicable)

### CODE REVIEW

**Comment by Wumpf** [comment]
does not check for accidental LFS files, just rooting for other stuff in CI breaking then ;)

**Review by emilk — APPROVED** [review]

**Review by jleibs — APPROVED** [review]
Any particular reason for doing this as bash instead of python?

**Comment by Wumpf** [comment]
> Any particular reason for doing this as bash instead of python?

Should have done python given that I spent soo much time figuring it out. Story is that I first had it in its own job and didn't want to install python. Then moved it into the util job which already has it but didn't convert it

File: scripts/check_large_files_allow_list.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+rerun_py/rerun_sdk/rerun_demo/demo.rrd
```
[the remark this answers is not in the document]
Comment (Wumpf) [reply scripts/check_large_files_allow_list.txt]: okay, odd. Emil removed it and it worked fine on ci. Okay whatever

File: scripts/check_large_files.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Whenever we want to make an exception, we add it to `check_large_files_allow_list.txt`
+set -eu
+
+# 300KiB maximum size.
+maximum_size=$((1024 * 300))
+
+result=0
+while read -d '' -r file; do
+    if [[ -f "$file" ]]; then
+        actualsize=$(wc -c <"$file")
```
[the remark this answers is not in the document]
Comment (Wumpf) [reply scripts/check_large_files.sh:12]: stat doesn't have that size filtering option on my mac it seems. I'll leave it be

[the remark this answers is not in the document]
Comment (emilk) [reply scripts/check_large_files.sh:12]: yet another reason to do this in Python

**Pull request merged**